content/en/observability_pipelines/guide/sensitive_data_scanner_transform.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Sensitive Data Scanner Transform
+kind: documentation
+disable_toc: false
+private: true
+is_beta: true
+further_reading:
+- link: "/observability_pipelines/setup/"
+  tag: "Documentation"
+  text: "Set up Observability Pipelines"
+- link: "/observability_pipelines/working_with_data/"
+  tag: "Documentation"
+  text: "Working with data in Observability Pipelines"
+---
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSfnNnV823zAgOCowCYuXJE5cDtRqIipKsYcNpaOo1LKpGfppA/viewform" btn_hidden="false" header="Request Access!">}}
+The <code>sensitive_data_scanner</code> transform is in private beta.
+{{< /callout >}}
+
+## Overview
+
+Sensitive data, such as credit card numbers, bank routing numbers, and API keys, can be revealed unintentionally in your logs, which can expose your organization to financial and privacy risks. Use the Observability Pipelines `sensitive_data_scanner` transform to identify, tag, and optionally redact or hash sensitive information before routing data to different destinations. You can use out-of-the-box scanning rules to detect common patterns such as email addresses, credit card numbers, API keys, authorization tokens, and more. Or, create custom scanning rules using regex patterns to match sensitive information.
+
+## Set up the `sensitive_data_scanner` transform
+
+1. Navigate to [Observability Pipelines][1].
+1. Click on your pipeline.
+1. Click **Edit as Draft**.
+1. Click **+ Add Component**.
+1. Select the **Transforms** tab.
+1. Click the **Sensitive Data Scanner** tile.
+1. Enter a name for the component.
+1. Select one or more inputs for the transform.
+1. Click **Add a New Item** to add a scanning rule, which determines what sensitive information to match within the data.
+1. Enter a name for the rule.
+1. In the **Define action on match** section, select the action you want to take for the matched information. Redaction, partial redaction, and hashing are all irreversible actions.  
```

Comment:
It may be worth describing the hash function used, in case users want to later search for that data (by re-hashing the original data themselves).

We hash the UTF-8 bytes of the match with the 64-bit fingerprint of farmhash.

Reply:
Thanks! Added the information.